docs/userguide.rst

```diff
@@ -185,8 +185,8 @@ the ``secrets-list`` command:
    $ reana-client secrets-list
    NAME                    TYPE
    .keytab                 file
-   HTCONDORCERN_KEYTAB     env
-   HTCONDORCERN_USERNAME   env
+   CERN_KEYTAB             env
+   CERN_USER               env
```

Comment:
Please add a section about Slurm in the user documentation, and mention that for both Slurm and HTCondor the workspace is transferred, and kept after the workflow finishes, so that people should clean it themselves...